.travis.yml

```diff
@@ -1,7 +1,10 @@
 language: node_js
-cache: yarn
+cache:  yarn
 node_js:
 - "8"
 install:
-- yarn install
-script: gitbook install && gitbook build
+  - yarn install
+  - yarn global add markdownlint-cli
```

Comment:
Could we move this to `package.json` under `devDependencies`? That way we would have all dependencies in one place.

Reply:
Is it possible to globally install it that way? Or if not, how to run the command from a local install?

Reply:
We do it for gitbook so I think we could do it for this also.

Reply:
Updated, it works!